share/wake/lib/system/remote_cache_api.wake

```diff
@@ -199,9 +199,18 @@ export def makeRemoteCacheApi (config: String): Result RemoteCacheApi Error =
     else failWithError "Remote cache config was set with non-integer port. Saw: {portStr}"
 
     def auth = if authStr ==* "" then None else Some authStr
+    def api = RemoteCacheApi domain port auth
 
-    RemoteCacheApi domain port auth
-    | Pass
+    # TODO: check for compatable wake version
```

Comment:
I thought this version check would be on the server side? What could we check for here?

Reply:
see follow PR :)